### ISSUE TITLE
RESTXQ: Handle reponse message and body

### ISSUE THREAD

**Issue by ChristianGruen** [BaseXdb/basex#1335]
If an HTTP response contains both a message and a body, the message will not be passed on. It needs to be checked if this is possible at all:

```xquery
declare %rest:path("x") function local:x() {
  <rest:response>
    <http:response status="404" message="ERROR!"/>
  </rest:response>,
  <xml/>
};
```

* Status: 404
* Message: Not Found (Standard)

**Comment by ChristianGruen** [comment]
The only clean solution seems to be the invocation of the deprecated [HTTPServletResponse.setStatus](https://tomcat.apache.org/tomcat-7.0-doc/servletapi/javax/servlet/http/HttpServletResponse.html#setStatus(int,%20java.lang.String)) function. There seems to be no other way to create both a status message (reason phrase) and a body.

**Comment by ChristianGruen** [comment]
Fixed in the `next` branch (will be made available with BaseX 8.6, as it changes the HTTP reponses of the REST and RESTXQ APIs).
